lib/elixir/lib/kernel.ex

```diff
@@ -975,8 +976,9 @@ defmodule Kernel do
       #=> %{b: 1}
 
   """
-  @spec tl(nonempty_maybe_improper_list(elem, tail)) ::
-        maybe_improper_list(elem, tail) | tail when elem: term, tail: term
+  @spec tl(nonempty_maybe_improper_list(elem, tail)) :: maybe_improper_list(elem, tail) | tail
+        when elem: term,
+             tail: term
```

Comment:
Note to self: figure out a way to put all entries on the same line after merged.